stats/opentelemetry/experimental/api.go

```diff
@@ -0,0 +1,21 @@
+package experimental
+
+import (
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// TraceOptions are the tracing options for OpenTelemetry instrumentation.
+//
+// # Experimental
```

Comment:
no need to mention experimental

Reply:
looks like we do in other places so let's keep it but now we are using experimental folder so may be not needed but its fine to keep